src/main/java/org/ngafid/MavenStarter.java

```diff
@@ -0,0 +1,73 @@
+package org.ngafid;
+
+import java.util.Map;
+
+import org.apache.maven.cli.MavenCli;
+
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class MavenStarter {
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+
+        MavenCli cli = new MavenCli();
+        //cli.doMain(new String[]{"clean", "install"}, "project_dir", System.out, System.out);
+        cli.doMain(new String[]{"exec:java", "-Dexec.mainClass='org.ngafid.ProcessUpload'"}, "/ngafid/ngafid2.0", System.out, System.out);
+
+
```

Comment:
Is there a reason for the commented out code to stay? If so, we might want a comment saying why it would be uncommented